client/app/telecom/telephony/alias/configuration/changeType/telecom-telephony-alias-configuration-changeType.html

```diff
@@ -1,104 +1,127 @@
     </div>
 
-    <div class="telecom-section-content"
-         data-ng-if="!AliasConfigurationChangeTypeCtrl.loading.line">
+    <div class="row">
+        <oui-collapsible class="col-lg-8 col-sm-12 ml-0 mr-2 mb-4"
+                         data-expanded="false"
+                         data-heading="{{:: 'telephony_alias_change_type_subtitle' | translate }}">
+            <p class="oui-paragraph"
+               data-translate="telephony_alias_change_type_details"></p>
+        </oui-collapsible>
+    </div>
 
-        <toast-message></toast-message>
+    <div class="form-group">
+        <oui-button class="ml-0"
+                    data-on-click="$ctrl.switchOptionMode()"
+                    data-variant="secondary">
+            <span data-translate="{{ ($ctrl.showExpertOptions ? 'telephony_alias_change_type_show_simple_option' : 'telephony_alias_change_type_show_expert_option') | translate }}"></span>
+        </oui-button>
+    </div>
 
-        <div class="alert alert-info"
-              role="alert">
-            <p data-translate="telephony_alias_change_type_info_text_1"></p>
-            <p data-translate="telephony_alias_change_type_info_text_2"></p>
+    <div data-ng-if="!$ctrl.showExpertOptions">
+        <div class="row w-100 d-md-flex">
+            <div class="col-lg-4 col-sm-12 mb-4">
+                <oui-select-picker class="h-100 pr-0 ml-0"
+                                   data-match="name"
+                                   data-model="$ctrl.selectedFeatureType"
+                                   data-label="{{:: 'telephony_alias_config_change_type_label_redirect' | translate }}"
+                                   data-values="[{ id: 'redirect', name: $ctrl.$translate.instant('telephony_alias_config_change_type_desc_redirect') }]"
+                                   data-variant="light">
+                    <span class="transclude-image-container">
+                        <img src="assets/images/aliasFeature/schema-redirect.svg"
+                             alt="schema-redirect.svg">
+                    </span>
+                </oui-select-picker>
+            </div>
+            <div class="col-lg-4 col-sm-12 mb-4">
+                <oui-select-picker class="h-100 pr-0"
+                                   data-match="name"
+                                   data-model="$ctrl.selectedFeatureType"
+                                   data-label="{{:: 'telephony_alias_config_change_type_label_contactCenterSolution' | translate }}"
+                                   data-values="[{ id: 'contactCenterSolution', name: $ctrl.$translate.instant('telephony_alias_config_change_type_desc_contactCenterSolution') }]"
+                                   data-variant="light">
+                    <span class="transclude-image-container">
+                        <img src="assets/images/aliasFeature/schema-easyHunting.svg"
+                             alt="schema-easyHunting.svg">
```

Comment:
Could be more descriptive?

Reply:
https://github.com/ovh-ux/ovh-manager-telecom/pull/981#discussion_r225066859